PipelinesTasks/PRMetrics/buildAndReleaseTask/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,156 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JamesNK/Newtonsoft.Json.Schema/master/Src/Newtonsoft.Json.Schema.Tests/Resources/Schemas/resjson.json",
+
+  "loc.description": "Updates the title, description, and comments of pull requests to provide at-a-glance metrics.",
```

Comment:
what do you mean when you say "at-a-glance metrics"?

Reply:
I've rewritten the description now, focusing on just the addition of the metrics to the title, which is really the key focus of the task.